system/lvol.py

```diff
@@ -119,7 +125,7 @@ def parse_lvs(data):
     for line in data.splitlines():
         parts = line.strip().split(';')
         lvs.append({
-            'name': parts[0],
+            'name': parts[0].replace('[','').replace(']',''),
```

Comment:
Could you please explain why do we need to remove [] letters? I've never saw them in lvs output.